infra/gcb/build_and_run_coverage.py

```diff
@@ -29,7 +31,18 @@
 
 # Where code coverage reports need to be uploaded to.
 COVERAGE_BUCKET_NAME = 'oss-fuzz-coverage'
-UPLOAD_URL_FORMAT= 'gs://%s/{project}/{type}/{date}' % COVERAGE_BUCKET_NAME
+
+# Link to the code coverage report in HTML format.
+HTML_REPORT_URL_FORMAT = (
+    '%s%s/{project}/reports/{date}/{platform}/index.html' % (
```

Comment:
mixing %s and {} is a little strange. Could you make this more consistent?

Reply:
Done in https://github.com/google/oss-fuzz/pull/1781